src/librustdoc/html/render.rs

```diff
@@ -112,6 +113,9 @@ struct Context {
     id_map: Rc<RefCell<IdMap>>,
     pub shared: Arc<SharedContext>,
     pub cache: Arc<Cache>,
+    pub parent: Rc<RefCell<Renderer>>,
```

Comment:
Just a note (for myself and any other potential reviewer): those 3 variables were before in the `run` function which became split in the `FormatRenderer` trait.